libraries/chain/wasm_interface.cpp

```diff
@@ -245,6 +262,11 @@ DEFINE_INTRINSIC_FUNCTION1(env,toUpper,toUpper,none,i32,charptr) {
 //   return 0;
 }
 
+DEFINE_INTRINSIC_FUNCTION1(env,loopControl,loopControl,i32,i32,i) {
+   usleep(50);
```

Comment:
What is this all about?    We shouldn't be sleeping.  I think this entire intrinsic can be removed.